src/main/kotlin/com/terraformation/backend/accelerator/db/VoteStore.kt

```diff
@@ -34,13 +35,27 @@ class VoteStore(
     }
   }
 
-  fun delete(projectId: ProjectId, phase: CohortPhase, userId: UserId? = null) {
+  fun fetchAllVoteDecisions(projectId: ProjectId): List<VoteDecisionModel> {
+    requirePermissions { readProjectVotes(projectId) }
+    return with(PROJECT_VOTE_DECISIONS) {
+      dslContext
+          .select(PROJECT_VOTE_DECISIONS.asterisk())
+          .from(PROJECT_VOTE_DECISIONS)
```

Comment:
This is totally fine as-is, but FYI, for single-table "select all columns" queries you can also do `selectFrom(PROJECT_VOTE_DECISIONS)` which selects all the columns and also causes the `fetch` method to return a `ProjectVoteDecisionsRecord` instead of a generic `Record`.

In this case since you're mapping the record to a custom class anyway, there's no functional difference (and doing it this way is 100% fine), just pointing out the feature so you'll know about it in the future.

For queries where you're selecting columns from more than one table, you have to use the `select(...).from(...)` style.